pyspark/bigdl/util/common.py

```diff
@@ -215,6 +257,34 @@ def from_ndarray(cls, features, label, bigdl_type="float"):
             label=JTensor.from_ndarray(label),
             bigdl_type=bigdl_type)
 
+    @classmethod
+    def from_jtensor(cls, features, label, bigdl_type="float"):
```

Comment:
refactor `from_ndarray` via `from_jtensor` to reduce the code duplication.

Reply:
Seems doesn't have duplication